src/app/modules/shared/utils/project-storage.util.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { updateProjectStorage, getProjectsOnStorage } from './project-storage.util';
+import { Project } from '../models/project.model';
+
+describe('Project Storage', () => {
+
+  let storageProjects: Project[];
+  let projectsIdentifier: string;
+  let serverProjects: Project[];
+
+  beforeEach(() => {
+    projectsIdentifier = 'projectsSelected';
+
+    storageProjects = [
+      {
+        customer: {
+          name: 'ioet Inc. (was E&Y)',
+        },
+        id: 'f3630e59-9408-497e-945b-848112bd5a44',
+        name: 'Time Tracker',
+        customer_id: '20c96c4d-5e26-4426-a704-8bdd98c83319',
+        status: 'active',
+      }
+    ];
+
+    serverProjects = [
+      {
+        customer: {
+          name: 'ioet Inc. (was E&Y)',
+        },
+        id: 'f3630e59-9408-497e-945b-848112bd5a44',
+        name: 'Time Tracker',
+        customer_id: '20c96c4d-5e26-4426-a704-8bdd98c83319',
+        status: 'active',
```

Comment:
when you mock something and it is repeating you must define a variable to avoid the repetition of values, for example here you are using the same values in two objects,

Reply:
Perfect, it was overlooked, now that is implemented.